avogadro/mainwindow.cpp

```diff
@@ -2192,7 +2192,11 @@ void MainWindow::buildTools()
   foreach (ToolPluginFactory* factory, toolPluginFactories) {
     ToolPlugin* tool = factory->createInstance(QCoreApplication::instance());
     tool->setParent(this);
-    tool->setIcon(darkMode);
```

Comment:
This makes no sense. `darkMode` here is a `bool` and you're using that in your conditional. If `setIcon()` does not exist (it does), your code won't work.

You've expanded one line into 4 but with the same logic.